packages/subapp-web/lib/load.js

```diff
@@ -193,10 +193,9 @@ Response: ${err || body}`
 
   const loadSubApp = () => {
     subApp = loadSubAppByName(name);
-    subAppServer = loadSubAppServerByName(name);
+    subAppServer = loadSubAppServerByName(name, true);
```

Comment:
Now we will call `loadSubapp` only if ssr is enabled. Setting second argument as `true` so that `loadSubAppServerByName` will generate server from subapp's main file if `serverEntry` is not there.